src/CommitmentsV2/SFA.DAS.CommitmentsV2/Mapping/Apprenticeships/ApprenticeshipToApprenticeshipDetailsMapper.cs

```diff
@@ -31,8 +31,9 @@ public Task<GetApprenticeshipsQueryResult.ApprenticeshipDetails> Map(Apprentices
                 EmployerName = source.Cohort.AccountLegalEntity.Name,
                 ProviderName = source.Cohort.Provider.Name,
                 StartDate = source.StartDate.GetValueOrDefault(),
-                EndDate = source.EndDate.GetValueOrDefault(),
+                EndDate = source.EndDate.GetValueOrDefault(),                
                 PauseDate = source.PauseDate.GetValueOrDefault(),
+                StopDate = source.StopDate.GetValueOrDefault(),
```

Comment:
Null is an allowable value

Reply:
PauseDate is also often null in the database - ive followed the same pattern